### PR TITLE
Add missing Restore-SPODataEncryptionPolicy SPO PS Cmdlet Documentation

### PR DESCRIPTION
**Pull request by jobrover**
Identity parameter was removed as it is no longer used.

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Restore-SPODataEncryptionPolicy.md [hunk 1]
```diff
@@ -14,49 +14,34 @@ ms.reviewer:
...
-Position: 1
-Default value: None
-Accept pipeline input: False
-Accept wildcard characters: False
-```
-
 ### -PrimaryKeyVaultName
 
-.
+The name of the primary key vault
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Restore-SPODataEncryptionPolicy.md:44]:
```suggestion
The name of the primary key vault.
```

Comment (get-itips) [reply sharepoint/sharepoint-ps/sharepoint-online/Restore-SPODataEncryptionPolicy.md:44]: please add the dot for the other descriptions, thanks.

**Review by dariomws — APPROVED** [review]
I'm ok after Andres suggestions. Thanks!

File: sharepoint/sharepoint-ps/sharepoint-online/Restore-SPODataEncryptionPolicy.md (hunk 1 above)
Comment (jobrover) [reply sharepoint/sharepoint-ps/sharepoint-online/Restore-SPODataEncryptionPolicy.md]: Done.

**Review by get-itips — APPROVED** [review]

**Pull request merged**